src/lib/db/event-store.ts

```diff
@@ -149,6 +149,16 @@ class EventStore implements IEventStore {
         }
     }
 
+    async getMaxRevisionId(largerThan: number = 0): Promise<number> {
+        const row = await this.db(TABLE)
+            .max('id')
+            .whereNotNull('feature_name')
+            .orWhere('type', 'segment-update')
```

Comment:
I just wonder what types of events to target (or which ones do we exclude?), if we ever expand and add new stuff we might forget to add another `orWhere` which might result in non-errors that would cause weird behaviors...

Reply:
I was tempted to say categorizing our event log with event types :D

Reply:
lets add that to the todo.